gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java

```diff
@@ -178,34 +167,14 @@ Logging createLoggingClient() {
   }
 
   @VisibleForTesting
-  static Map<String, String> getCustomTags(Map<String, String> customTags,
-      Map<String, String> locationTags, String projectId) {
+  static Map<String, String> getCustomTags(Map<String, String> customTags) {
```

Comment:
This function effectively does very little/nothing - handle `null` on line 172 may be. Can you just inline this function?

Reply:
The function also creates an immutable map of `customTags`. On further looking realized `null` check is not required, as [ObservabilityConfigImpl](https://github.com/grpc/grpc-java/blob/master/gcp-observability/src/main/java/io/grpc/gcp/observability/ObservabilityConfigImpl.java#L120-L133) will always return `Collections.emptyMap()` when custom tags are not set in the configuration.

So I have two options:
1. Remove `null` check and keep the function as-is to create an immutable map.
2. Make the method inline and remove `null` check i.e
```
this.customTags = config.getCustomTags();
```

I prefer Option 1, as it gives us an immutable map. Lmk what you think.

Reply:
Okay sounds good- option 1 is ok